index.bs

```diff
@@ -24,6 +24,18 @@ Mailing List Archives: http://lists.w3.org/Archives/Public/public-webappsec/
 
 Markup Shorthands: css no, markdown yes
 </pre>
+
+<pre class="biblio">
+{
```

Comment:
this should live here: https://github.com/WICG/admin/blob/gh-pages/biblio.json

Reply:
(it will be automatically ingested by SpecRef + BikeShed in 24 hours)

Reply:
Thanks @marcoscaceres! I'll send a PR to update the WICG/admin code.